python/semantic_kernel/connectors/ai/open_ai/services/azure_chat_completion.py

```diff
@@ -208,10 +209,12 @@ def __init__(
         if base_url and isinstance(base_url, str):
             base_url = HttpsUrl(base_url)
         if use_extensions and endpoint and deployment_name:
-            base_url = HttpsUrl(f"{str(endpoint).rstrip('/')}/openai/deployments/{deployment_name}/extensions")
+            base_url = HttpsUrl(
```

Comment:
I think we can get rid of the extensions flag (and allow base_url for the people still using that endpoint)? @moonbox3 WDYT?

Reply:
That seems fine with me.